spring-cloud-skipper-server-core/src/test/java/org/springframework/cloud/skipper/server/service/ArgumentSanitizerTests.java

```diff
@@ -36,7 +36,7 @@ public void testNoChange() throws Exception {
 				TestResourceUtils.qualifiedResource(getClass(), "nopassword.yaml").getInputStream(),
 				Charset.defaultCharset());
 		String result = ArgumentSanitizer.sanitizeYml(initialYaml);
-		assertThat(result).isEqualTo(initialYaml);
+		assertThat(result.replace("\r", "")).isEqualTo(initialYaml.replace("\r", ""));
```

Comment:
In general, this makes me worry that we will play wak-a-mole throughout other code finding places like this. 

1. Is there something we can change in the underlying saved files to avoid having to special case the code? 

2. If we can't fix in the file, wdyt about moving this to a utility method? (`assertEqualsIgnoringXYZ()`)